dali/test/python/test_operator_crop.py

```diff
@@ -254,6 +254,8 @@ def __init__(self, device, batch_size, iterator, data_shape, data_layout, num_th
             D, H, W = self.data_shape[0], self.data_shape[1], self.data_shape[2]
         elif self.data_layout == "CDHW":
             D, H, W = self.data_shape[1], self.data_shape[2], self.data_shape[3]
+        elif self.data_layout == "FHWC" and crop_seq_as_depth:
```

Comment:
Is that possible to have "FCHW" as well?

Reply:
Done